addons/auth_password_policy_signup/static/src/js/signup_policy.js

```diff
@@ -1,25 +1,23 @@
 odoo.define('auth_password_policy_signup.policy', function (require) {
```

Comment:
Commit message fail:
- 72 characters
- Wrong targeting , portal has changed (and should be explained)